catalogue/webapp/pages/work.js

```diff
@@ -282,8 +282,8 @@ export const WorkPage = ({
                     <div>
                       <b>Locations</b>
                       <ul>
-                        {item.locations.map(location => (
-                          <div key={location.id}>
+                        {item.locations.map((location, i) => (
+                          <div key={`location${i}`}>
```

Comment:
How much do we care about the idea that ['index as a key is an anti-pattern'](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)?

Inconceivable that it would bite us here.

Reply:
Here, not so much, but this is prototypey